server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -57,9 +57,18 @@ object LivyConf {
   val SUPERUSERS = Entry("livy.superusers", null)
 
   val AUTH_TYPE = Entry("livy.server.auth.type", null)
-  val KERBEROS_PRINCIPAL = Entry("livy.server.auth.kerberos.principal", null)
-  val KERBEROS_KEYTAB = Entry("livy.server.auth.kerberos.keytab", null)
-  val KERBEROS_NAME_RULES = Entry("livy.server.auth.kerberos.name_rules", "DEFAULT")
+  val AUTH_KERBEROS_PRINCIPAL = Entry("livy.server.auth.kerberos.principal", null)
+  val AUTH_KERBEROS_KEYTAB = Entry("livy.server.auth.kerberos.keytab", null)
+  val AUTH_KERBEROS_NAME_RULES = Entry("livy.server.auth.kerberos.name_rules", "DEFAULT")
+
+  val LAUNCH_KERBEROS_PRINCIPAL =
+    LivyConf.Entry("livy.server.launch.kerberos.principal", null)
+  val LAUNCH_KERBEROS_KEYTAB =
+    LivyConf.Entry("livy.server.launch.kerberos.keytab", null)
+  val LAUNCH_KERBEROS_REFRESH_INTERVAL =
+    LivyConf.Entry("livy.server.launch.kerberos.refresh_interval_seconds", 3600)
```

Comment:
minor: can you use "1h" as the default, and read the config using `getTimeAsMs`? That's more user-friendly.